starknet/src/utils/simple_quorum.cairo

```diff
@@ -33,10 +34,9 @@ mod SimpleQuorumExecutionStrategy {
             votes_against: u256,
             votes_abstain: u256,
         ) -> ProposalStatus {
-            let accepted = InternalImpl::quorum_reached(
-                self._quorum.read(), votes_for, votes_against, votes_abstain
-            )
-                & InternalImpl::supported(votes_for, votes_against);
+            let quorum_reached = votes_for + votes_abstain >= self._quorum.read();
```

Comment:
we do no count the `votes_against` anymore? I believe this differs from the previous implem?

Reply:
Actually, I'm surprised no test broke ^^ 
we should have a test with only `AGAINST` so that it reaches quorum but with no `abstain` or `for`

Reply:
Nope we dont add the against votes, same as [sx-evm](https://github.com/snapshot-labs/sx-evm/blob/a17a4abbc8a35276163fbbb051bf14c879980b96/src/execution-strategies/SimpleQuorumExecutionStrategy.sol#L72). 

Otherwise it would create an adverse incentive for those voting against to just not vote at all.